include/aspect/particle/world.h

```diff
@@ -111,6 +111,18 @@ namespace aspect
         const Particles::ParticleHandler<dim> &
         get_particle_handler() const;
 
+        /**
+         * Copy the state of particle handler @p from_particle_handler into the
+         * particle handler @p to_particle_handler. This will copy
+         * all particles and properties and leave @p to_particle_handler
+         * as an identical copy of @p from_particle_handler, assuming it
+         * was set up by this particle world class. This means we assume
+         * @p from_particle_handler uses the same triangulation and
+         * particle properties as are used in this model.
+         */
```

Comment:
can you comment about what happens with existing particles in ``to_particle_handler``?